cmake/CMakeLists.txt

```diff
@@ -1288,34 +1288,6 @@ if (onnxruntime_USE_OPENVINO)
 
   add_definitions(-DUSE_OPENVINO=1)
 
-  if (EXISTS "$ENV{INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/version.txt")
-    file(READ $ENV{INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/version.txt VER)
-  endif()
-
-  if (NOT DEFINED ENV{INTEL_OPENVINO_DIR})
```

Comment:
This ensures whether OPENVINO is sourced into the env. Not advisable to remove as it might end up with crash

Reply:
`INTEL_OPENVINO_DIR` is not defined in many distribution types like conda-forge, brew etc.
It's defined only by OpenVINO archive.

In order to check OpenVINO version, you must either use `OpenVINO_VERSION` vars from `OpenVINOConfig.cmake` or OpenVINO headers (`openvino/core/version.hpp`). 

Please, see other changes where I've added version check in C++ files.